overlays.nix

```diff
@@ -0,0 +1,20 @@
+final: prev:
+
+let
+  inherit (prev.stdenv) isAarch32 isAarch64 isDarwin;
+
+  darwinSpecific = if isAarch64 then {
+    inherit (prev.darwin.apple_sdk_11_0.frameworks) Accelerate MetalKit;
+  } else {
+    inherit (prev.darwin.apple_sdk.frameworks) Accelerate CoreGraphics CoreVideo;
+  };
```

Comment:
Nit: could do

```
apple_sdk = if ... then prev.darwin.apple_sdk_11_0 else prev.darwin_apple_sdk;
inherit (apple_sdk) ...;
```

(on that note, I'm not sure why do we pass these as individual arguments instead of passing `apple_sdk` as the whole thing; outside my expertise though)

Reply:
Mine too. I'm keeping it from the previous impl, but I don't really [understand the structure](https://github.com/NixOS/nixpkgs/blob/master/pkgs/os-specific/darwin/apple-sdk/frameworks.nix#L2).